components/backdrop/backdrop.js

```diff
@@ -123,6 +124,7 @@ class Backdrop extends LitElement {
 			if (!reduceMotion && !this.noAnimateHide && isVisible(this)) {
 				this.addEventListener('transitionend', hide, { once: true });
 				this._state = 'hiding';
+				setTimeout(() => this._state === 'hiding' && hide(), TRANSITION_DURATION + 100);
```

Comment:
So was `transitionend` just not firing in all cases?

Reply:
Whoops, thought I filled out the description more. Yeah it just doesn't fire, which is really weird. As far as I can tell `transitioncancel` also doesn't fire. The only thing I can think is there's some weird combo of low-CPU but also hardware acceleration going on where the timing just gets weird.

Reply:
Gotcha. There are likely a few other places where we do this, but they probably don't have such dire consequences if they're missed.